KIF Tests/AutocorrectTests.m

```diff
@@ -47,8 +47,8 @@ - (void)afterEach
 - (void)testSmartQuotesEnabled
 {
     if (@available(iOS 11.0, *)) {
-        [tester clearTextFromAndThenEnterText:@"'\"'," intoViewWithAccessibilityLabel:@"Greeting" traits:UIAccessibilityTraitNone expectedResult:@"’”’,"];
-    }
+        [[viewTester usingLabel:@"Greeting"] clearAndEnterText:@"'\"'," expectedResult:@"’”’,"];
```

Comment:
There is another test class `AutocorrectTests_ViewTestActor.m` for testing viewTester to ensure parity between the two. Needing to maintain two interfaces is a pain, would be good to deprecate the old syntax.

Reply:
Created a ticket here: https://github.com/kif-framework/KIF/issues/1176